tests/test_pool.py

```diff
@@ -17,7 +17,7 @@ class FakeXYZBackend(ThreadSafeCreateMixin):
 
 class TestBackendPool(object):
 
-    def test_get_backends_should_return_all_backends_instances(self, settings):
```

Comment:
I tried this approach, but I got some issues. Can I try this in another moment? 🤔